src/test/README.md

```diff
@@ -29,6 +30,7 @@ To add more unit tests, add `BOOST_AUTO_TEST_CASE` functions to the existing
 implement new `BOOST_AUTO_TEST_SUITE` sections.
 
 To run the GUI unit tests manually, launch `build/src/qt/test/test_bitcoin-qt`
+(assuming `build` is your build directory).
```

Comment:
Same here. I don't think we need to duplicate `(assuming build is your build directory)` 10 lines apart. Especially when all the build commands here used `build` as the build dir.